src/lwc/rd2ElevateInformation/rd2ElevateInformation.html

```diff
@@ -88,8 +88,8 @@ <h2 slot="title" class="slds-p-top_xx-small" data-qa-locator={qaLocatorHeader}>
                                     for="commitmentId" id="commitmentIdLabel">{fields.commitmentId.label}</label>
                                 <div class="slds-form-element__control slds-p-around_none" data-id="commitmentId">
                                     <div class="slds-col slds-no-flex slds-truncate">
-                                        <a href="javascript:void(0);" aria-labelledby="commitmentIdLabel"
-                                            target="_blank">
+                                        <a href={commitmentURL} aria-labelledby="commitmentIdLabel"
```

Comment:
Do you think that it's worth considering using the LWC [NavigationService](https://developer.salesforce.com/docs/component-library/documentation/en/lwc/lwc.use_navigate) for this versus `a href`?

Reply:
I looked at this and decided it was a poor fit for what we are doing.  I don't believe it really adds anything other than additional code complexity.